webui/ocw/lib/EC2db.py

```diff
@@ -35,8 +35,8 @@ def _instance_to_json(i):
 @transaction.atomic
 def sync_instances_db(region, instances):
     o = Instance.objects
-    o = o.filter(region=region, provider=ProviderChoice.EC2, state=StateChoice.ACTIVE)
-    o = o.update(state=StateChoice.UNK, active=False)
+    o = o.filter(region=region, provider=ProviderChoice.EC2)
```

Comment:
how this relates to GCE ? if not can you please explain what is happening here ?

Reply:
This is really not directly related to GCE, but I figured that out during GCE extending.
The thing is that the synchronization between local DB and the CSP status was a bit obfuscated.

It works now like:
  * Instance.active is used to identify if such entry exists on CSP
  * Instance.state is used to reflect the latest "local" state, e.g. if delete button was pressed.

Reply:
can you please add this to field description ?

Reply:
as comments where you declare Instance ( db.py )